barrels/Semicircles/.settings/IQ_IDE.prefs

```diff
@@ -0,0 +1,4 @@
+eclipse.preferences.version=1
```

Comment:
Do we want to submit this?

Reply:
Should I remove the entire .settings directory or just the IQ_IDE.prefs file?

Reply:
Go ahead and leave out the whole file ... I think this a local eclipse plugin artifact

Reply:
The settings file here is similar to the .project in that it's specifying project specific settings in regards to Eclipse. In this case the file shouldn't be necessary because it only contains default values but there's no harm in having the file checked in.